content/blog/automated-releases-part-two.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Full automation of release with GitHub Actions and Conventional Commits for non-JS projects - part 2"
```

Comment:
What about this?

```suggestion
title: "(Part 2) Full automation of release with GitHub Actions and Conventional Commits for non-JS projects"
```

It looks like you're writing the second part of this blog post but you're not. The other one wasn't for non-JS projects. Instead, this is the second part of a series of blog posts and this is the title. What do you think?